### ISSUE TITLE
Sticker packs for Telegram/Signal/Element

### ISSUE THREAD

**Issue by funnelfiasco** [SeaGL/organization#185]
For the stickers, what license do we want them to be under? CC BY 4.0? CC BY-SA 4.0? CC0? I assume it won't be a problem with whoever we select because it's work for hire, but I want to make sure up front.

We also need to tell them what we want. I have a few ideas, would like to collect a few more:

* Patch flying with wings outstretched
* Patch with a French fry
* Patch drinking tea
* Patch hacking on a computer
* Patch with a Magic 8 Ball (as a nod to last year's speaker gift)
* Patch stretched out relaxing

Anything else we can come up with?

**Comment by altsalt** [comment]
I think we can give them all of those options but generally prefer/suggest CC BY-SA 4.0

In terms of ideas, I know that they often make use of a certain number of "emotions" or "reactions" that map to the smiley faces you'd expect. Unsure if that's helpful, but may be a partial guide.

**Comment by mateuskrause** [comment]
Small things:

- What is a good name for the sticker pack? "SeaGL", "Path the Seagull", "Seagull Path"... Personally, I like those who use the name "Path". 
- I'll assign some emojis for the stickers (like salt commented here) and this we can change later if necessary.

**Comment by altsalt** [comment]
How about "Patch the SeaGL Seagull"?

Thanks for working on this!

**Comment by mateuskrause** [comment]
Stickers update!

For now, the sticker pack is live on Telegram and Signal:
- [Patch the SeaGL Seagull Telegram](https://t.me/addstickers/PatchSeagull)
- [Patch the SeaGL Seagull Signal](https://signal.art/addstickers/#pack_id=37da6dc461d100aef3eb1fd134aaa363&pack_key=82badb0ba139c941cc6e89e4cb8dc2d20f60ed858877553ca95f7b71c0f23722)

For Element, some things: The sticker pack was created (and we can host at seagl.org repo), but for adding we need (each user) to manually add an event, inserting a json in developer tools. The good thing is that after this, they are available on mobile, computer, web...

I found that we can use [Dimension](https://github.com/turt2live/matrix-dimension) and in theory, will be easier to share the pack, but still not available in mobile and will be more complicated to configure.

What do you think, _stick_ with the current way and write one small tutorial for how to add the sticker pack?

**Comment by altsalt** [comment]
Hmm, did you have to ascribe emoji or descriptions to each image? I'm doing the Dimension process and it's asking for that... I'll just put something in if there isn't a standard thing yet.

**Comment by altsalt** [comment]
https://stickers.t2bot.io/pack/@salt:seattlematrix.org/xtaZRNpnoh-Patch-the-SeaGL-Sea

01: Patch the SeaGL seagull flying.
02: Patch the SeaGL seagull lounging.
03: Patch the SeaGL seagull pointing.
04: Patch the SeaGL seagull is concerned.
05: Patch the SeaGL seagull being silly.
06: Patch the SeaGL seagull on a laptop.
07: Patch the SeaGL seagull eating fries.
08: Patch the SeaGL seagull drinking tea.
09: Patch the SeaGL seagull singing.

**Comment by mateuskrause** [comment]
Yes, I had to assign the emojis, the ones I choose was:

01: Patch the SeaGL seagull flying: 😐
02: Patch the SeaGL seagull lounging: 😎
03: Patch the SeaGL seagull pointing: 😊
04: Patch the SeaGL seagull is concerned: 😩/😟
05: Patch the SeaGL seagull being silly:  😃
06: Patch the SeaGL seagull on a laptop: 😌/💻
07: Patch the SeaGL seagull eating fries: 😋/🍟
08: Patch the SeaGL seagull drinking tea: 🙃/🍵
09: Patch the SeaGL seagull singing: 😄

Probably in some stickers I didn't choose the best option (choosing emojis is not my strong point 😶 ) so let me know and I'll change in Telegram and Signal

**Comment by mateuskrause** [comment]
I'm in the making of the interstitials (#174) and will be good to create a shorter url for the sticker pack, like _seagl.org/stickers_ or something related. @funnelfiasco it's possible to add one redirect to the blog post? The `redirect_from` works on the posts?

**Comment by altsalt** [comment]
It should, but if not, you can make a file like this:
https://github.com/SeaGL/seagl.github.io/blob/main/feedback.md?plain=1

**Comment by mateuskrause** [comment]
The `redirect_from` worked. I did a [pull request](https://github.com/SeaGL/seagl.github.io/pull/301).

**Comment by altsalt** [comment]
I GOT IT WORKING!!!

Also, updated the sticker names to include emoji (changed a couple if you want to update yours @mateuskrause ) as seen here:
https://github.com/SeaGL/seagl.github.io/tree/main/stickerpicker/Patch%20the%20SeaGL%20seagull

To enable them in element-web:
1) go to any room where you can access and enter the `Room settings`
2) navigate to `Advanced` and click on `Open Devtools`
3) click on `Send Account Data`
4) set the Event Type to `m.widgets` and the Event Content to:
```
{
    "stickerpicker": {
        "content": {
            "type": "m.stickerpicker",
            "url": "https://seagl.org/stickerpicker/web/?theme=$theme",
            "name": "Stickerpicker",
            "data": {}
        },
        "sender": "@you:matrix.server.name",
        "state_key": "stickerpicker",
        "type": "m.widget",
        "id": "stickerpicker"
    }
}
```
(note to change the sender to your account information)
5) click `Send` and close the Devtools
6) add stickers to any room via the `Show Stickers` icon!

**Comment by altsalt** [comment]
Btw, wanted to mention that we should add the SeaGL logo to these packs. Not a huge rush on it, maybe something to do with the next run and such, but would be nice.

**Comment by mateuskrause** [comment]
Yes, this makes total sense actually. I'll create a version of the logo with a white border like on the stickers.

**Comment by koronkowy** [comment]
Stickers happened, woo!

**Comment by altsalt** [comment]
It did, but we're not quite done as we'll be adding the logo to the packs and potentially a few more stickers. Thanks for going through and cleaning up though!
